src/LocalDateRange.php

```diff
@@ -228,6 +228,16 @@ public function count() : int
         return $this->end->toEpochDay() - $this->start->toEpochDay() + 1;
     }
 
+    /**
+     * Returns Duration of this range
+     *
+     * @return Duration
+     */
+    public function toDuration(): Duration
+    {
+        return Duration::ofDays($this->count());
```

Comment:
Shouldn't this be `Duration::ofDays($this->count() - 1)`?

As it stands, `toPeriod()` returns a 0-day Period if the start date is equal to the end date (this is in line with its Java counterpart).

So wouldn't it be weird if it returned a 1-day Duration for, say, `2022-06-05/2022-06-05`?

Reply:
> This object is iterable and countable: the iterator returns all the LocalDate objects contained
> in the range, while `count()` returns the total number of dates contained in the range.
 
> The end date, inclusive.

I think duration of one day range is One day, not zero)

Reply:
>As it stands, toPeriod() returns a 0-day Period if the start date is equal to the end date (this is in line with its Java counterpart).

Quite weird. Is LocalDateRange in java inclusive too?

Reply:
I think it makes sense in terms of Period: there is `1 day` between `2022-06-06` an `2022-06-07`.
In terms of Duration, maybe the approach is different. But this can be confusing.